doc/geany.txt

```diff
@@ -5560,10 +5560,14 @@ Editor
 * Ctrl-click on a word in a document to perform *Go to Symbol Definition*.
 * Ctrl-click on a bracket/brace to perform *Go to Matching Brace*.
 
-Interface
----------
+Sidebar
+-------
 
-* Double-click on a symbol-list group to expand or compact it.
+* Symbol list - double-click on a symbol category to expand or compact it (or click the arrow).
```

Comment:
```suggestion
* Double-click on an item in the symbols or documents lists to expand or fold its children.
```

It's actually a treeview feature (I believe) and it works in the symbols list, the documents list, the keybindings groups in the prefs, etc.  Anywhere the item has children and the double-click action isn't overriden I guess.

Reply:
Yes, updated. I listed it under GTK.